src/Init/Data/List/Basic.lean

```diff
@@ -1001,6 +1004,9 @@ def IsSuffix (l₁ : List α) (l₂ : List α) : Prop := Exists fun t => t ++ l
 
 @[inherit_doc] infixl:50 " <:+ " => IsSuffix
 
+/-- not `isSuffix` -/
+recommended_spelling "suffix" for "<:+" in [IsSuffix, «term_<:+_»]
```

Comment:
This goes against the community naming convention. Will there be a discussion about recommended spellings?

Reply:
Does it? There seems to be barely anything in mathlib about `List.IsSuffix`. https://loogle.lean-lang.org/?q=List.IsSuffix

Reply:
More generally, I have made an attempt to always use the established naming. If you think I made a mistake somewhere, feel free to note it here or start a thread on Zulip about specific spellings.

Reply:
`suffix` is the current name, but it goes against the naming convention (which would instead be `isSuffix`). I am worried that this `recommended_spelling` will turn the statu quo into an official recommendation.

Reply:
I don't think it is a problem if a notation has a different name than the declaration it refers to. For example, we say `singleton` for `[x]` and not something like `consNil`.